backend/src/main/kotlin/app/ehrenamtskarte/backend/mail/Mailer.kt

```diff
@@ -28,18 +39,142 @@ object Mailer {
+        project: String,
+        backendConfig: BackendConfiguration,
+        projectConfig: ProjectConfig
+    ) {
+        val recipients = AdministratorsRepository.getNotificationRecipientsForApplication(project)
+        val message = """
+        Guten Tag,
+
+        Ein neuer Antrag liegt in ${projectConfig.administrationName} vor.
+        Sie können neue Anträge direkt unter ${projectConfig.administrationBaseUrl}/applications einsehen und bearbeiten.
+
+        Falls Sie keine weiteren Benachrichtigungen zu neuen Anträgen erhalten möchten können Sie dies unter ${projectConfig.administrationBaseUrl}/user-settings deaktivieren.
+
+        Dies ist eine automatisierte Nachricht. Antworten Sie nicht auf diese Email.
+        
+        - ${projectConfig.administrationName}
+        """.trimIndent()
+        for (recipient: AdministratorEntity in recipients) {
+            try {
+                sendMail(
+                    backendConfig,
+                    projectConfig.smtp,
+                    projectConfig.administrationName,
+                    recipient.email,
+                    "Ein neuer Antrag ist eingegangen",
+                    message
+                )
+            } catch (_: MailException) {}
         }
-        MailerBuilder.withSMTPServer(smtpConfig.host, smtpConfig.port, smtpConfig.username, smtpConfig.password)
-            .buildMailer()
-            .sendMail(
-                EmailBuilder
-                    .startingBlank()
-                    .to(to)
-                    .from(fromName, smtpConfig.username)
-                    .withSubject(subject)
-                    .withPlainText(message)
-                    .withHeader("Content-Type", "text/plain; charset=utf8")
-                    .buildEmail()
-            ).join()
+    }
+
+    fun sendNotificationForVerificationMails(
+        project: String,
+        backendConfig: BackendConfiguration,
+        projectConfig: ProjectConfig
+    ) {
+        val recipients = AdministratorsRepository.getNotificationRecipientsForVerification(project)
+        val message = """
+        Guten Tag,
+
+        Ein ein Antrag ist Verifiziert worden. 
+        Sie können Anträge direkt unter ${projectConfig.administrationBaseUrl}/applications einsehen und bearbeiten.
+
+        Falls Sie keine weiteren Benachrichtigungen zu neuen Anträgen erhalten möchten können Sie dies unter ${projectConfig.administrationBaseUrl}/user-settings deaktivieren.
+
+        Dies ist eine automatisierte Nachricht. Antworten Sie nicht auf diese Email.
+        
+        - ${projectConfig.administrationName}
+        """.trimIndent()
+        for (recipient: AdministratorEntity in recipients) {
+            try {
+                sendMail(
+                    backendConfig,
+                    projectConfig.smtp,
+                    projectConfig.administrationName,
+                    recipient.email,
+                    "Ein neuer Antrag ist eingegangen",
```

Comment:
```suggestion
                    "Ein neuer Antrag wurde verifiziert",
```
Falscher Betreff, wenn ein neuer Antrag verifiziert wurde